### DISCUSSION TITLE
Are Game Server Login Tokens (GSLT, +sv_setsteamaccount) still required with CS2?

### DISCUSSION THREAD

**Discussion by dasisdormax** [dasisdormax/cs2-multiserver#26] [General]
I found conflicting information about this on the internet.

a) They are required, just as with CS:GO before.
b) They are not required, and the game will use the steam account from SteamCMD.

In my personal testing, they were never required, but I only hosted servers in the local network.

Someone please test an online game server without a GSLT, and report their results.

**Comment by mrc4tt** [comment]
You won't need GSLT. (Only if you're planning to run weaponpaints plugin on the CounterStrikeSharp addon)

**Comment by pretsell** [comment]
Setting GSLT will let your server be displayed in the server browser.

To generate code: https://steamcommunity.com/dev/managegameservers
